TreeDBConnect.php

```diff
@@ -73,4 +73,25 @@ public function sortAsc(): array
 
         return $trees;
     }
+
+    public function addNewTree(Tree $newTree): void
+    {
+        $sqlstatement = 'INSERT INTO `trees` (`name`
+        ,`scientific_name`
+        ,`leaf_type`
+        ,`leaf_colour`
+        ,`flower_colour`
+        ,`image`)
+        VALUES (:name,:scientific_name,:leaf_type,:leaf_colour,:flower_colour,:image);';
+
+        $values = ['name' => $newTree->getName()
+            , 'scientific_name' => $newTree->getScientificName()
+            , 'leaf_type' => $newTree->getLeafType()
+            , 'leaf_colour' => $newTree->getLeafColour()
+            , 'flower_colour' => $newTree->getFlowerColour()
+            , 'image' => $newTree->getImageLink()];
```

Comment:
Very nitpicky, but we usually put the comma on the end of the line when breaking down arrays onto multi line like so:

```
$thing = [
    'value',
    'another'
];
```

Reply:
I've changed these for you